### DISCUSSION TITLE
哔哩 Wiki 还未建设

### DISCUSSION THREAD

**Discussion by Cyenoch** [Richasy/Bili.Uwp#68] [问答]
无法进入到Wiki页面，是目前还没有内容吗

**Comment by luojunyuan** [comment]
点击进去就返回主页，应该是还没搞。构建运行直接clone 切换到平台vs就能跑起来

**Reply by Cyenoch** [reply]
已经是跑起来了，效果很好

**Comment by Richasy** [comment]
由于之前一直在闭源开发，Wiki不支持在private项目中开启。现在刚开源，还没有建设

**Reply by Cyenoch** [reply]
静候建设完成，有好多地方没能看懂。
